src/back/services/events.ts

```diff
@@ -336,7 +336,10 @@ export class EventsService {
       created_at: new Date(discoursePost.created_at),
     }
 
-    return await EventModel.create(commentedEvent)
+    await EventModel.create(commentedEvent)
+    NotificationService.newCommentOnProjectUpdate(commentedEvent)
+    DclNotificationService.newCommentOnProjectUpdate(commentedEvent)
```

Comment:
I'm worried that there is a growing amount of repetition between this two.

I'd rather have one notification service, and treat `DclNotificationService` as we do with any other third party service. 
Preferably, It should not have direct access to our models, but rather be provided with the necessary data by our `NotificationService`

Maybe we could do something like this:

```typescript
static newCommentOnProjectUpdate(commentEvent: ProjectUpdateCommentedEvent) {
    inBackground(async () => {
      const proposalId = commentEvent.event_data.proposal_id
      const updateId = commentEvent.event_data.update_id
      try {
        const proposal = await ProposalModel.getProposal(proposalId)
        const coauthors = await CoauthorService.getAllFromProposalId(proposalId)
        const coauthorsAddresses = coauthors.length > 0 ? coauthors.map((coauthor) => coauthor.address) : []
        const recipients = [proposal.user, ...coauthorsAddresses]

        DclNotificationService.newCommentOnProjectUpdate(proposalData, recipients)
        DiscordService.sendToRecipients(proposalData, recipients) //checks internally for active users
        return await this.sendNotification(notification, recipients)

      } catch (error) {
        // ...
      }
    })
  }
  ```

Reply:
I mean, 

```
NotificationService.projectProposalEnacted(proposal)
DclNotificationService.projectProposalEnacted(proposal)

NotificationService.coAuthorRequested(newProposal, coAuthors)
DclNotificationService.coAuthorRequested(newProposal, coAuthors)

NotificationService.sendFinishProposalNotifications(proposalsWithOutcome)
DclNotificationService.sendFinishProposalNotifications(proposalsWithOutcome)

etc
```

They both make the same queries and are called exactly at the same places.

Reply:
Will be addressed in an upcoming PR!